distributed/core.py

```diff
@@ -832,6 +832,9 @@ def __init__(
         self._created = weakref.WeakSet()
         self._instances.add(self)
 
+    def validate(self):
```

Comment:
Since this is only used for testing, I'd make this `_validate`, and add a docstring for it explaining what it tests.

Reply:
Fixed it